.travis.yml

```diff
@@ -2,31 +2,36 @@ sudo: true
 dist: trusty
 language: node_js
 node_js:
-  - 'stable'
+  - '6.7.0'
   - '4.2.1'
+addons:
+  firefox: "latest"
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-beta
 before_install:
   - 'npm config get registry'
   - 'npm config set registry http://registry.npmjs.org/'
-  - 'export CHROME_BIN=/usr/bin/google-chrome'
   - 'export DISPLAY=:99.0'
   - 'sh -e /etc/init.d/xvfb start'
-  - 'sudo apt-get update'
-  - 'sudo apt-get install -y libappindicator1 fonts-liberation'
-  - 'wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb'
-  - 'sudo dpkg -i google-chrome*.deb'
+  - 'sleep 3'
```

Comment:
It was recommended by Travis
https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI